x/subspaces/simulation/genesis.go

```diff
@@ -17,11 +16,11 @@ func RandomizeGenState(simState *module.SimulationState) {
 
 // randomSubspaces returns randomly generated genesis account
 func randomSubspaces(simState *module.SimulationState) (subspaces []types.Subspace) {
-	subspacesNumber := simState.Rand.Intn(100)
-
-	subspaces = make([]types.Subspace, subspacesNumber)
-	for index := 0; index < subspacesNumber; index++ {
+	subspaces = make([]types.Subspace, len(subspaces))
+	for index := 0; index < len(subspaces); index++ {
```

Comment:
The initial length of subspaces is zero so that no subspace is created. I think it should be reverted.

Reply:
Good catch @dadamu. I actually wanted to use the Len of the provided `ids` array inside the utils and I wrongly use the returned var. I will fix this now